test/libsolidity/syntaxTests/pragma/valid_range_operator.sol

```diff
@@ -0,0 +1,2 @@
+pragma solidity 0.8.17 - 0.8.20;
```

Comment:
For positive cases please use a wider range of versions. This one will break in 0.8.21.

Other tests we already have use at least 0.9.0, but I think it would not hurt to use something much higher so that updating these tests is less annoying when we reach a breaking release.